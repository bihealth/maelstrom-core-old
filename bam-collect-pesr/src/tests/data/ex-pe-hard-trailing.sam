../../../../bam-scan/src/tests/data/ex-pe-hard-trailing.sam