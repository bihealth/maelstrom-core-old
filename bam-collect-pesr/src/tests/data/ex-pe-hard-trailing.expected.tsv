#contig	start	end	signal
one	299	349	{"SplitRead":{"read_name":"pair_expected","is_first":false,"contig":"one","start":299,"end":349,"clipped_sides":"Right"}}
two	100099	100149	{"SplitRead":{"read_name":"pair_expected","is_first":false,"contig":"two","start":100099,"end":100149,"clipped_sides":"Right"}}
two	100099	100149	{"PairedRead":{"read_name":"pair_expected","is_first1":false,"contig1":"two","start1":100099,"end1":100149,"strand1":"Reverse","contig2":null,"start2":null,"strand2":null,"tlen":null}}
